live/clean.sh

```diff
@@ -11,36 +11,41 @@ cleanPrefixes() {
     for p in ${prefixes[@]}; do
         echo "Cleaning prefix $p"
         cleanTerraFiles
-        for region in $(ls); do
-             if [[ $region != "account.hcl" ]]; then
-                echo "Clearing k8 auth from state for $region"
-                pushd $region
-                for env in ${envs[@]}; do
-                    echo "Clearing $env"
-                    pushd $env/physical
-                    TG_STATE_PREFIX="test/$p/" terragrunt state rm "module.eks_cluster.kubernetes_config_map.aws_auth[0]"
-                    popd
-                done
-                popd
-             fi
-        done
+#         for region in $(ls); do
+#              if [[ $region != "account.hcl" ]]; then
+#                 echo "Clearing k8 auth from state for $region"
+#                 pushd $region
+#                 for env in ${envs[@]}; do
+#                     echo "Clearing $env"
+#                     pushd $env/physical
+#                     TG_STATE_PREFIX="test/$p/" terragrunt state rm "module.eks_cluster.kubernetes_config_map.aws_auth[0]"
+#                     popd
+#                 done
+#                 popd
+#              fi
+#         done
```

Comment:
Temporarily commented out?

Reply:
More like it's there as an option if you want to enable it. Sometimes it's necessary but it adds quite a bit of time to the clean script so I usually have it commented out unless I need it.